independent-projects/tools/common/src/main/java/io/quarkus/cli/commands/ListExtensions.java

```diff
@@ -66,8 +67,14 @@ public void listExtensions(boolean all, String format, String search) throws IOE
                 System.out.println("\nTo get more information, append -Dquarkus.extension.format=full to your command line.");
```

Comment:
I may be wrong, but I think this instruction doesn’t really work in Gradle. Can you verify this?

Reply:
you are right; i missed that.